doc/CCPPtechnical/source/CompliantPhysicsParams.rst

```diff
@@ -59,16 +59,16 @@ General Rules
 =============
 A CCPP-compliant scheme is in the form of Fortran modules. :ref:`Listing 2.1 <scheme_template>` contains
 the template for a CCPP-compliant scheme (``ccpp/framework/doc/DevelopersGuide/scheme_template.F90``),
-which includes four essential components: argument metadata tables, the *_init*, *_run*, and *_finalize*
-subroutines.
+which includes three essential components: the *_init*, *_run*, and *_finalize* subroutines. Each ``.f`` or ``.F90``
```

Comment:
This description leaves off the "breadcrumb" for doxygen to print the argument table. We're still requiring that schemes have
`!> \section arg_table_schemename_run Argument Table
!! \htmlinclude schemename_run.html
!!`

Reply:
Good catch. I have made this change in the doc/DevelopersGuide/scheme_template.F90 so it is now reflected in the TechDoc. I also added a blurb explaining this to both DevelopersGuide/scheme_template.F90 and doc/CCPPtechnical/source/CompliantPhysicsParams.rst.